pandas/io/formats/latex.py

```diff
@@ -41,9 +41,9 @@ def __init__(
         self.multirow = multirow
         self.clinebuf: List[List[int]] = []
         self.strcols = self._get_strcols()
-        self.strrows: List[List[str]] = (
-            list(zip(*self.strcols))  # type: ignore[arg-type]
-        )
+        self.strrows: List[List[str]] = list(
+            zip(*self.strcols)
+        )  # type: ignore[arg-type]
```

Comment:
```suggestion
            zip(*self.strcols)  # type: ignore[arg-type]
        )
```

Reply:
thanks :+1: that solves the typing validation

I'll @ pandas/core / pandas/triage , do you think the rest of the contributors need alerting? I don't think there's that many other people who regularly submit PRs

Reply:
I think pandas/core and pandas/triage is probably fine